server/server.js

```diff
@@ -1,40 +1,52 @@
+// importing needed modules
 import express from "express";
 import cors from "cors";
 import "dotenv/config";
+import https from "https";
+import path from "path";
+import { fileURLToPath } from "url";
+import fs from "fs";
+// importing custom sequelize class
 import databaseHelper from "./databasehelper.js";
 
+// create express app
 const APP = express();
+APP.use(express.json());
+APP.use(cors());
 
-// dev environment vars
-// const DATABASE = process.env.DATABASE;
-// const USERNAME = process.env.USERNAME;
-// const PASSWORD = process.env.PASSWORD;
-// const PORT = 5000;
-
-// prod environment vars
-// const DATABASE = process.env.PGDATABASE;
-// const USERNAME = process.env.USER;
-// const PASSWORD = process.env.PGPASSWORD;
-const PORT = process.env.PGPORT;
-
-// const dbhelper = new databaseHelper(DATABASE, USERNAME, PASSWORD);
+// get directory name
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
-const dbhelper = new databaseHelper();
+// current app environment
+const APP_ENV = process.env.APP_ENV;
+console.log("App environment:", APP_ENV);
 
-try {
-  await dbhelper.db.authenticate()
-} catch (err) {
-  console.error('Unable to connect to the database:', err)
+// dev environment vars
+export const DATABASE = process.env.DATABASE;
+export const USERNAME = process.env.USERNAME;
+export const PASSWORD = process.env.PASSWORD;
+export const PORT = process.env.PORT;
+
+// creates a sequelize instance
+const dbhelper = new databaseHelper(APP_ENV);
+
+// pull cat & dog breeds from api, add them to preset list of breeds, and insert them into breeds table
+function populateBreedsTable() {
+  dbhelper.getCatBreeds();
+  dbhelper.getDogBreeds();
 }
 
-dbhelper.db.sync();
-
-dbhelper.createSamplePet();
-dbhelper.getCatBreeds();
-dbhelper.getDogBreeds();
+populateBreedsTable();
 
-APP.use(express.json());
-APP.use(cors());
+// function to alphabetize breed lists by name before sending to client
+function alphabetize(a, b) {
```

Comment:
Is this more accurately called `alphabetizeByName`?

Reply:
It is indeed! I was staring at it wondering why on earth I'd rewrite sort to do exactly what it does by default, realized what I was doing, left the comment, and simply did not think to rename it. Fixed!